src/Analyzers/MSTest.Analyzers/Resources.resx

```diff
@@ -543,4 +543,10 @@ The type declaring these methods should also respect the following rules:
   <data name="DynamicDataShouldBeValidMessageFormat_SourceTypeNotPropertyOrMethod" xml:space="preserve">
     <value>'[DynamicData]' member '{0}.{1}' is not a property nor a method. Only properties and methods are supported.</value>
   </data>
-</root>
+  <data name="UseNewerAssertThrowsTitle" xml:space="preserve">
+    <value>Use newer methods to assert exceptions</value>
```

Comment:
I don't recall, do we see title on the menu or message?

Reply:
I think the message. Not sure of all places that show the title, but Solution Explorer is one that shows the title (when you expand Dependencies -> Analyzers)